apps/server/src/controllers/integrationController.ts

```diff
@@ -174,6 +174,19 @@ export function dispatchFromAdapter(
       PlaybackService.roll();
       break;
     }
+    case 'addtime': {
+      const time = Number(payload);
+      if (isNaN(time)) {
+        throw new Error(`Time not recognised ${payload}`);
+      }
+      try {
+        PlaybackService.addTime(time);
+      } catch (error) {
+        throw new Error(`Could not add time: ${error}`);
+      }
```

Comment:
I see this is a pattern we use here, we should make a todo to improve this. In this case, re-throwing the error does not add any extra information or value